index.js

```diff
@@ -1,4 +1,4 @@
-// import your node modules
+// import your node modules 
 
 const db = require('./data/db.js');
 const server = require('express')();
```

Comment:
Some people don't like it when you curry a require because it can look a little confusing (but that's one of the first things I tried out when we got into express, so I understand). Later on, you'll probably want an express variable just because you'll use properties on it for your server code (specifically, you'll be wanting to use it once we get into middleware).